.github/workflows/pr.yaml

```diff
@@ -37,6 +37,7 @@ jobs:
       - unit-tests-cudf-pandas
       - pandas-tests
       - pandas-tests-diff
+      - third-party-integration-tests-cudf-pandas
```

Comment:
Ignore the CI changes. I added them to show the third-party integration tests were passing. I'll delete them once this PR is approved.

Reply:
Sorry for the ping